### PR TITLE
JSON output

### LINKED ISSUE #16: json output
Since in clair.go the function analyse returns a json, I would like to ask if it is possible to extend main.go from line 60 to 67 in order to let the user decide if she wants a json output or a human readable text.

A json output will integrate klar in many solutions since the results can be showed in any fashion you like.

### PR DESCRIPTION
**Pull request by palfrey**
Fixes #16

Gives the option of JSON output for all the current outputted info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ Default is `Unknown`.
 
 * `DOCKER_PASSWORD` - Docker registry account password
 
+* `KLAR_INSECURE` - Allow Klar to access registries with bad SSL certificates. Default is `false`.
```
Comment (supereagle) [inline README.md]: It is better to remove the code about insecure registry, and this PR should only focus on json output.

File: main.go [hunk 2]
```diff
@@ -64,26 +72,46 @@ func main() {
 		fmt.Printf("Can't pull image: %s", err)
 		os.Exit(1)
 	}
+
+	var useJSONOutput = false
```
Comment (supereagle) [inline main.go]: The params reading should be along with others.

File: README.md (hunk 1 above)
Comment (palfrey) [inline README.md]: Done, now that #19 is merged and I've rebased this

File: main.go (hunk 2 above)
Comment (palfrey) [inline main.go]: Done

File: main.go [hunk 3]
```diff
@@ -68,26 +80,41 @@ func main() {
...
+			fmt.Printf("Analysing %d layers\n", len(image.FsLayers))
+		}
 	}
 
 	c := clair.NewClair(clairAddr)
 	vs := c.Analyse(image)
 	groupBySeverity(vs)
-	fmt.Printf("Found %d vulnerabilities \n", len(vs))
+	if !useJSONOutput {
+		fmt.Printf("Found %d vulnerabilities \n", len(vs))
```
Comment (hashmap) [inline main.go]: what if we move this line to line 110 to reduce numbers of ifs?

Comment (palfrey) [inline main.go]: Sounds sensible, done

**Comment by hashmap** [comment]
Great job, thanks!

**Pull request merged**